libraries/SITL/SIM_Scrimmage.h

```diff
@@ -0,0 +1,93 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/*
+  simulator connection for Scrimmage Simulator
+*/
+
+#pragma once
+
+#include <string>
+#include <map>
+
+#include <AP_HAL/utility/Socket.h>
+
+#include "SIM_Aircraft.h"
+
+namespace SITL {
+
+/*
+  a last_letter simulator
+ */
+class Scrimmage : public Aircraft {
+public:
+    Scrimmage(const char *home_str, const char *frame_str);
+
+    /* update model by one time step */
+    void update(const struct sitl_input &input);
```

Comment:
missing 'override' on this method is why the CI is failing

Reply:
fixed